### ISSUE TITLE
[Rated Disabilities] v3 component migration

### ISSUE THREAD

**Issue by jacobworrell** [department-of-veterans-affairs/va.gov-team#76124]
## Background 

We need to migrate v1 components to v3 components. There are 6 components that need to be migrated to V3 in the Letters app. We need to remove the false prop for 14 components. We can beginning working this ticket after 2/16.

Here is a link to the [audit doc](https://docs.google.com/spreadsheets/d/1ajKjN6dr-2PzpgEbfX1OqxAupvbqoHwqh7ml1m5a1Fo/edit?usp=sharing). 

## Steps

- [ ] Any components that were updated as USWDS false that are not V3 components should have that prop removed.
- [ ] Design and engineering test locally to confirm design fidelity of the migration before changes are merged

## Acceptance Criteria

- [ ] All V1 components with a V3 version are migrated to that version of the component.
- [ ] All V1 components that do not have a V3 version do not have a prop set to false
- [ ] Post migration, the UI meets the BMT team's design and Ux standards.

**Comment by HeatherWidmont** [comment]
Peri hasn't seen any messages about a merge that was supposed to happen on 3/4. @pmclaren19 will check if there have been any announcements, but as of right now this is still blocked.

**Comment by pmclaren19** [comment]
On March 4th Design team announced this would now happen on 4/11/24.

**Comment by pmclaren19** [comment]
Was Blocked until https://github.com/department-of-veterans-affairs/vets-website/pull/28486 was merged. they merged on 3/15.

**Comment by pmclaren19** [comment]
@jacobworrell please validate when you can. Take a look at the pr that was merged for more details!

**Comment by HeatherWidmont** [comment]
Jacob took a look & said this should probably be validated by Design instead. @uxgary will take a look

**Comment by uxgary** [comment]
Peri and I have been testing this morning, we found a VA link that was added to the project with "USWDS=false". Peri will update this in another PR. However, as we looked for other test accounts to find users with alerts, staging has been having issues. We can't successfully log in with any users, including the one we've signed in with earlier. 

Note: We tested user log in on our respective machines to make sure it wasn't an isolated it. Peri sent a message to support.
